workflow-templates/im-deploy-on-prem-website.yml

```diff
@@ -69,16 +69,16 @@ jobs:
     runs-on: ubuntu-latest # Force this to run on github-hosted runner by using a tag that does not exist on self-hosted runners
 
     outputs:
-      IIS_Server: ${{ steps.set-variables.outputs.IIS_SERVER }} # To use this value: ${{ needs.set-vars.outputs.IIS_SERVER }}
+      ON_PREM_WINDOWS_SERVER: ${{ steps.set-variables.outputs.ON_PREM_WINDOWS_SERVER }} # To use this value: ${{ needs.set-vars.outputs.ON_PREM_WINDOWS_SERVER }}
```

Comment:
This workflow won't only be for on-prem machines. It could be for Azure VMs as well.

I'm thinking the original name `IIS_Server` isn't that bad.